src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -238,37 +239,33 @@ void LocalResourceManager::FreeTaskResourceInstances(
   }
 }
 
-std::vector<double> LocalResourceManager::AddResourceInstances(
-    scheduling::ResourceID resource_id, const std::vector<double> &resource_instances) {
+std::vector<double> LocalResourceManager::AddCpuForTempBlockedWorker(
+    const std::vector<double> &resource_instances) {
   std::vector<FixedPoint> resource_instances_fp =
       FixedPointVectorFromDouble(resource_instances);
 
+  auto resource_id = ResourceID::CPU();
   if (resource_instances.size() == 0) {
     return resource_instances;  // No overflow.
   }
 
-  bool is_idle = true;
   auto overflow =
       AddAvailableResourceInstances(resource_instances_fp,
                                     local_resources_.total.GetMutable(resource_id),
                                     local_resources_.available.GetMutable(resource_id),
-                                    &is_idle);
+                                    nullptr);
 
-  if (is_idle) {
```

Comment:
This is the major change for blocked worker cpu resource tracking: 
- we don't let temporary resources allocation change from blocked/unblocked worker to affect the resource idleness

Reply:
cc @jjyao